File: hw/1126/1126.cpp
#include <iostream>
using namespace std;

int main() {
    setlocale(LC_ALL, "");
    int arr[] = {3, -5, 8, 0, -2, 7, -4};
    int size = sizeof(arr) / sizeof(arr[0]);

    // Выводим неотрицательные элементы
    cout << "Неотрицательные элементы:";
    for (int i = 0; i < size; i++) {
        if (arr[i] >= 0) {
            cout << " " << arr[i];
        }
    }
    cout << endl;

    // Выводим отрицательные элементы
    cout << "Отрицательные элементы:";
    for (int i = 0; i < size; i++) {
        if (arr[i] < 0) {
            cout << " " << arr[i];
        }
    }
    cout << endl;

    return 0;
}

File: hw/1126/1126.py
arr = [3, -5, 8, 0, -2, 7, -4]

# Выводим неотрицательные элементы
print("Неотрицательные элементы:", end=' ')
for num in arr:
    if num >= 0:
        print(num, end=' ')

print()

# Выводим отрицательные элементы
print("Отрицательные элементы:", end=' ')
for num in arr:
    if num < 0:
        print(num, end=' ')
